src/app/workbasket/workbasket.component.ts

```diff
@@ -83,7 +83,7 @@ export class WorkbasketComponent implements OnInit {
     let paginationMetadataObservable = this.paginationService
       .getPaginationMetadata(filter.jurisdiction.id, filter.caseType.id, metadataPaginationParams, caseFilters);
 
-    Observable.forkJoin(searchObservable, paginationMetadataObservable)
+    forkJoin(searchObservable, paginationMetadataObservable)
```

Comment:
That change is not related to the POC and is a good change to bring in the code base

Reply:
Sure will bring it in.